modules/ROOT/pages/test-ssl-enabled-sources-cookbook.adoc

```diff
@@ -0,0 +1,139 @@
+= Testing SSL-Enabled Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To test an application that has an HTTP Listener with SSL configured, you must configure an SSL truststore in your test to send authenticated requests to your code using a TLS certificate.
+
+== Example
```

Comment:
I suggest omitting this heading and using the scenario as the topic set intro, and then moving to the two testing methods.
```suggestion
```